local/local.go

```diff
@@ -608,7 +620,7 @@ func inspectOptionalImage(docker client.CommonAPIClient, imageName string) (type
 
 	if inspect, _, err = docker.ImageInspectWithRaw(context.Background(), imageName); err != nil {
 		if client.IsErrNotFound(err) {
-			return defaultInspect(), nil
+			return types.ImageInspect{}, nil
```

Comment:
Since the signature for `defaultInspect` changed from `defaultInspect() ImageInspect` to `defaultInspect(dockerClient) (ImageInspect, error)` (with an extra query), I wasn't sure it was worth the extra code for what I (probably naively) assumed was empty result. Happy to add it back though.